app/services/flow/flows/provider_start.rb

```diff
@@ -48,6 +48,10 @@ class ProviderStart < FlowSteps
           path: ->(application) { urls.providers_legal_aid_application_about_the_financial_assessment_path(application) },
           forward: :application_confirmations
         },
+        update_email_addresses: {
+          path: ->(application) { urls.providers_legal_aid_application_update_email_address_path(application) },
+          forward: :application_confirmations
```

Comment:
According to the Jira ticket, `forward` should be `about_the_financial_assessments`